packages/quantic/force-app/examples/main/lwc/exampleQuanticGeneratedAnswer/exampleQuanticGeneratedAnswer.html

```diff
@@ -20,15 +20,79 @@
       use-case={config.useCase}
       engine-id={engineId}
     >
-      <c-quantic-generated-answer
-        engine-id={engineId}
-        fields-to-include-in-citations={config.fieldsToIncludeInCitations}
-        collapsible={config.collapsible}
-        max-collapsed-height={config.maxCollapsedHeight}
-        with-toggle={config.withToggle}
-        answer-configuration-id={config.answerConfigurationId}
+      <div
+        class="slds-grid slds-gutters_direct slds-wrap main slds-grid_align-center"
       >
-      </c-quantic-generated-answer>
+        <template lwc:if={config.withFacets}>
+          <div
+            class="slds-col slds-order_2 slds-large-order_1 slds-size_1-of-1 slds-large-size_6-of-12"
+          >
+            <c-quantic-facet-manager engine-id={engineId}>
```

Comment:
I see you are only using timeframe facets in your test suit, why then having all the facets here?